ArjunaJTA/recovery/src/main/java/org/jboss/narayana/jta/quickstarts/recovery/JmsRecovery.java

```diff
@@ -299,66 +276,15 @@ public int consumeMessages(int cnt, long millis) throws Exception {
             endJTATx(true);
 
             xaConsumer.close();
-            xaSession.close();
-        }
-        finally
-        {
-             connection.close();
         }
 
         return msgCnt;
     }
 
     public void drainQueue() throws Exception {
         while (consumeMessages(100, 500) == 100)
-           System.out.println("drained 100 messages");
-    }
-
-    public void testIsJmsWorking() throws Exception {
```

Comment:
this method is never called

Reply:
Yep, I see your point here. My only concern is whether this is a functional or a style change. I'd lean toward functional here but that's up to you

Reply:
Thanks @jmfinelli for your review, I will then include it in a separate commit called 'Cleaning the code'